src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, Arial;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12px;
+}
+
+header {
+  margin: 10px 0 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo-link {
+  text-align: center;
+}
+
+a.is-active {
+  color: #00acdc;
+  text-decoration: underline;
+  position: relative;
+}
+
+.is-active::after {
+  content:"";
+  position: absolute;
+  top: 30px;
+  left: 0;
+  width: 37px;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+}
```

Comment:
The `is-active` class and its pseudo-element `::after` seem to be doing the same thing - changing the link color and adding an underline. This is a bit redundant and can be simplified. You could achieve the same effect by just changing the link color and using the `text-decoration` property in the `is-active` class.